### PR TITLE
Implementation of the Logo component

### PR DESCRIPTION
**Pull request by hcafaq**
### Description

Describe your changes (or add ticket description) and add relevant screenshots

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change

### How do I test this

- Add steps to test
- in bullet point format
- preferably you can add link to the storybook build in the PR

### Did you remember to take care of the following?

- [ ] `npm i` – for new NPM dependencies.
- [ ] `npm run lint` - to check for linting issues
- [ ] `npm run test` - to run unit tests
- [ ] `npm run test:screenshots` - to run snapshot tests

### New Feature / Bug Fix

- [ ] Run unit tests to ensure all existing tests are still passing.
- [ ] Add new passing unit tests to cover the code introduced by your pr.

Thanks for contributing!

### CODE REVIEW

File: packages/logo/src/Logo.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <svg
+      aria-hidden={ariaHidden}
+      aria-label={ariaLabel}
+      className={classNames()}
+      fill="none"
+      height={size}
+      role={role}
+      viewBox="0 0 199 65"
+      width={size * 2}
+      xmlns="http://www.w3.org/2000/svg"
```
[the remark this answers is not in the document]
Comment (derek-heycar) [reply packages/logo/src/Logo.tsx:26]:
```
<svg
       styles={{ fontSize }}
        aria-hidden={ariaHidden}
        aria-label={ariaLabel}
        className={classNames()}
        role={role}
        viewBox="0 0 199 65"
        xmlns="http://www.w3.org/2000/svg"
```

**Review by derek-heycar — CHANGES_REQUESTED** [review]

File: packages/logo/src/Logo.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hcafaq) [reply packages/logo/src/Logo.tsx]: done

File: packages/logo/src/docs/Component.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    },
+  }}
+  argTypes={{
+    fontSize: {
+      control: 'number',
+    },
+    color: {
+      control: {
+        type: 'select',
+        options: ['primary', 'secondary', 'tertiary'],
```
Comment (derek-heycar) [inline packages/logo/src/docs/Component.stories.mdx:39]: add please default value `primary `

File: packages/logo/src/docs/Component.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  args={{
+    color: 'primary',
+    fontSize: 200,
+  }}
+  parameters={{
+    backgrounds: {
+      values: [
+        { name: 'primary', value: '#052962' },
+        { name: 'secondary', value: '#FFFFFF' },
+        { name: 'tertiarys', value: '#F1F4FA' },
```
Comment (derek-heycar) [inline packages/logo/src/docs/Component.stories.mdx:28]: do not need to replace color for the whole canvas. Could you please do the similar as for the documentation part?
justthe  wrap logo to the square div, and change the background of the div depending on the chosen color of the logo.

**Review by derek-heycar — CHANGES_REQUESTED** [review]

File: .storybook/main.js [hunk 4]
```diff
@@ -30,13 +30,14 @@ module.exports = {
...
   addons: [
-    '@storybook/addon-controls',
     '@storybook/preset-create-react-app',
     {
       name: '@storybook/addon-docs',
       options: { transcludeMarkdown: true },
     },
     'storybook-addon-live-examples',
+    '@storybook/addon-controls',
+    '@storybook/addon-backgrounds',
```
Comment (derek-heycar) [inline .storybook/main.js:40]: it's not needs

File: packages/logo/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+Heycar Mint = Secondary
+Sunbeam Blue = Tertiary
+
+for more info you checkout colour page in [Figma](https://www.figma.com/file/rrvkvQEoVTOHa7MzyKSaoz/OTP-Design-System?node-id=1291%3A35195)
+
+##installation and usage
+
+import Logo from '@heycar-uikit/logo';
+
+<Logo fontSize={65} color="primary" />
```
Comment (derek-heycar) [inline packages/logo/README.md:20]: this section better move to the beginning of the file.
do not forget add link and the storybook.
and wrap please code part of the document to the special code quotes ```` `

**Review by derek-heycar — CHANGES_REQUESTED** [review]

File: packages/logo/src/docs/Description.mdx [hunk 6]
```diff
@@ -0,0 +1,48 @@
+import { Container, Row, Col } from 'storybook/blocks';
+import Logo from '../Logo';
+
+## Colors
+
+Use the `color` prop to change the visual color of the Logo.
+You can set the value to `primary`, `secondary` , `tertiary`
+
```
Comment (emrahsemizheycar) [inline packages/logo/src/docs/Description.mdx:8]: add following line here : 
```tsx live

File: packages/logo/src/docs/Description.mdx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+          backgroundColor: '#F1F4FA',
+          padding: '15px',
+          textAlign: 'center',
+        }}
+      >
+        <Logo fontSize={150} color="tertiary" />
+      </div>
+    </Col>
+  </Row>
+</Container>
```
Comment (emrahsemizheycar) [inline packages/logo/src/docs/Description.mdx:48]: and after this line add following line : 
\```

File: packages/logo/src/docs/Description.mdx (hunk 6 above)
Comment (emrahsemizheycar) [reply packages/logo/src/docs/Description.mdx:8]: you can check an example here : https://raw.githubusercontent.com/hey-car/heycar-uikit/main/packages/button/src/docs/Description.mdx

File: packages/logo/package.json [hunk 8]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@heycar-uikit/logo",
+  "version": "1.0.0",
+  "description": "HeyCar-UIKit logo",
+  "main": "dist/index.js",
+  "module": "./dist/esm/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (emrahsemizheycar) [inline packages/logo/package.json:8]: Just out of curiosity , why do we have this test script saying no test specified ? in a package that actually has tests inside. This is not the only component with this line in its package.json. Some components have this line some don't . any reason ? @derek-heycar I also wonder your opinion on this.

Comment (hcafaq) [reply packages/logo/package.json:8]: I think we can actually remove this as we have specified jest for the main package json

File: packages/logo/src/docs/Description.mdx (hunk 6 above)
Comment (hcafaq) [reply packages/logo/src/docs/Description.mdx:8]: Yes it was live earlier but after conversation with @derek-heycar we decided we didn't need it for the logo

File: .storybook/main.js (hunk 4 above)
Comment (hcafaq) [reply .storybook/main.js]: resolved

File: packages/logo/README.md (hunk 5 above)
Comment (hcafaq) [reply packages/logo/README.md]: resolved

**Pull request merged**